pkg/service/types.go

```diff
@@ -27,7 +28,11 @@ type MeshService struct {
 	// If the service resides on a Kubernetes service, this would be the Kubernetes namespace.
 	Namespace string
 
-	// The name of the service
+	// The name of the service. May include instance (e.g. pod) information if the backing service
```

Comment:
Please add an example for the Name value when this represents a non-headless k8s service, as it is unclear whether the subdomain will be set.